### PR TITLE
Add LocationLookup component

### PR DESCRIPTION
**Pull request by StellaHuSEEK**
Screen Shot 2022-05-11 at 4 35 13 pm

### CODE REVIEW

File: fe/lib/components/LocationLookup/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Installation
+
+```shell
+yarn add wingman-fe
+```
+
+## Location Lookup Widget
+
+The LocationLookup widget abstracts the `location` query on SEEK API and provides a select input that returns a SEEK location.
```
Comment (72636c) [inline fe/lib/components/LocationLookup/README.md:13]:
```suggestion
The LocationLookup widget abstracts the `location` query on SEEK API and provides a text input that returns a SEEK location.
```

File: fe/lib/components/LocationLookup/LocationLookup.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  debounceDelay?: number;
+}
+
+export const LocationLookup = ({
+  schemeId,
+  initialLocationId,
+  debounceDelay = 250,
+  client,
+}: LocationLookupProps) => {
+  const [locationId, setLocationId] = useState(initialLocationId ?? '');
```
Comment (72636c) [inline fe/lib/components/LocationLookup/LocationLookup.tsx:36]: Do we need a `useEffect` to re-`setLocationId` if `initialLocationId` changes? A scenario I'm thinking of is propagating an ID from query params or local storage to this component, but the ID is (mistakenly?) not available on the first render.

File: fe/lib/components/LocationLookup/LocationLookup.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          }
+        />
+      </Stack>
+
+      {locationLoading && <Loader />}
+
+      {locationError && (
+        <FieldMessage
+          id="locationLookupError"
+          message="Sorry, we couldn’t fetch the location. Please try again."
```
Comment (72636c) [inline fe/lib/components/LocationLookup/LocationLookup.tsx:84]:
```suggestion
          message="Sorry, we couldn’t retrieve this location. Please try again."
```

**Review by 72636c — APPROVED** [review]

**Review by 72636c — APPROVED** [review]
🚀

File: fe/lib/components/LocationLookup/LocationLookup.tsx (hunk 2 above)
Comment (StellaHuSEEK) [reply fe/lib/components/LocationLookup/LocationLookup.tsx:36]: Tested locally that `initialLocationId` is set on the first-time rendering. Also we can add the useEffect() function later if any valid scenario need it

**Pull request merged**